Docs/source/usage/how_to_run.rst

```diff
@@ -49,19 +49,28 @@ Copy the **executable** to this directory:
 
 where ``<warpx_executable>`` should be replaced by the actual name of the executable (see above) and ``<run_directory>`` by the actual path to the run directory.
 
-3. Inputs
----------
+Input File
+----------
+
+You need to provide WarpX with an input file that configures the simulation.
+This can either be a parameter list or a Python script, depending on how you wish to run WarpX.
+
+To run the WarpX executable, add a **parameter list** file in the directory (see :ref:`examples <usage-examples>` and :ref:`parameters <running-cpp-parameters>`).
+This is a text file containing the numerical and physical parameters that define the simulation.
 
-Add an **input file** in the directory (see :ref:`examples <usage-examples>` and :ref:`parameters <running-cpp-parameters>`).
-This file contains the numerical and physical parameters that define the situation to be simulated.
+To run WarpX through the Python interface, add a **PICMI Python script** (see :ref:`examples <usage-examples>` and :ref:`PICMI parameters <usage-picmi-parameters>`).
+This is a Python script that defines the numerical and physical parameters using the `PICMI standard <https://picmi-standard.org/>`__.
 
-On :ref:`HPC systems <install-hpc>`, also copy and adjust a submission script that allocated computing nodes for you.
+On :ref:`HPC systems <install-hpc>`, also copy and adjust a submission script that allocates computing nodes for you.
 Please :ref:`reach out to us <contact>` if you need help setting up a template that runs with ideal performance.
 
-4. Run
-------
+Run Simulation
+--------------
 
-**Run** the executable, e.g. with MPI:
+WarpX Executable
```

Comment:
I would use a Sphinx design tab here, because it is either executable or script -- not both serially in a sensible workflow.

https://impactx.readthedocs.io/en/latest/usage/how_to_run.html#how-to-run